content/posts/newsletter-015/index.md

```diff
@@ -60,6 +60,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### Worship The Sun
+
+![Worship The Sun](worship_the_sun.jpg)
+_▲ One of the many unique and confounding puzzles in the game_
```

Comment:
Not sure if the arrow is necessary here, as the website applies formatting to the image captions to make it more obvious they're connected to the image.